src/containers/tutor-home-page/general-info-step/GeneralInfoStep.jsx

```diff
@@ -1,21 +1,188 @@
-import { Typography } from '@mui/material'
-import Box from '@mui/material/Box'
+import { useEffect, useState } from 'react'
+import { Typography, Box, Grid } from '@mui/material'
 import { useTranslation } from 'react-i18next'
-
+import AppTextField from '~/components/app-text-field/AppTextField'
+import AppTextArea from '~/components/app-text-area/AppTextArea'
 import { styles } from '~/containers/tutor-home-page/general-info-step/GeneralInfoStep.styles'
+import illustration from '~/assets/img/tutor-home-page/become-tutor/general-info.svg'
+import { createFilterOptions } from '@mui/material'
+import AppAutoComplete from '~/components/app-auto-complete/AppAutoComplete'
+import axios from 'axios'
+import { useStepContext } from '~/context/step-context'
+
+const COUNTRIES_API_URL = 'https://restcountries.com/v3.1/all'
+const CITIES_API_URL = 'http://api.geonames.org/searchJSON'
```

Comment:
Do you think it would be better to move these API URL to the .env file?

Reply:
Yeah, it's much better. Update it